src/components/QueryResult.vue

```diff
@@ -70,12 +79,38 @@ export default Vue.extend( {
 		color: $font-color-subtle;
 		font-weight: $font-weight-style-description;
 		line-height: $font-line-height-style-description;
+		width: $dimension-width-full;
+		border: $border-width-thin $border-style-base $border-color-base-subtle;
+		border-radius: 0 0 $border-radius-base $border-radius-base;
 		text-align: center;
 	}
 
+	.querybuilder__result__header {
+		display: flex;
+		align-items: center;
+		background: $background-color-neutral-default;
```

Comment:
```suggestion
		background: $color-base-90;
```

Reply:
I believe that we should use a global token here, since that "neutral" background is semantically related to the "flavor" of some components (e.g. buttons). The decision can be revised in the event that this header is turned into a component.

Reply:
That was mentioned in the spec but if that's what you think should be, I'm all fine with it.